Flutter/pdf-viewer/page-layout-and-scroll-direction.md

```diff
@@ -54,7 +54,7 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
-![Single page layout mode](images/page-layout-and-scroll-direction/page-by-page.gif)
+![Single page layout mode in Flutter PDF Viewer.](images/page-layout-and-scroll-direction/flutter-pdf-viewer-page-by-page.gif)
```

Comment:
Is this image available in the folder?
<img width="771" alt="image" src="https://github.com/syncfusion-content/flutter-docs/assets/61148953/072fdff5-ddda-46a1-9cfd-3e50e265bf2b">

Reply:
resolved